docs/oauth2-oidc/wellknown-endpoint-discovery.mdx

```diff
@@ -0,0 +1,834 @@
+---
+id: wellknown-endpoint-discovery
+title: OAuth2.0 & OpenID Connect well-known endpoint discovery
+sidebar_label: OAuth2.0 & OpenID Connect well-known endpoint discovery
+---
+
+There are two types of well-known discovery endpoints. The OAuth2.0 and the OpenID Connect (ODIC) discovery endpoints. Both
+endpoints return nearly identical information as OIDC is built on top of OAuth 2.0 (OIDC is a superset of OAuth 2.0).
+
+The discovery endpoints provide a standardized way for clients to automatically discover your server's configuration and
+capabilities. They eliminate manual configuration, enable dynamic multi-tenant support, and make OAuth 2.0 and ODIC integrations
+more resilient to changes. Discovery endpoints allow clients to adapt automatically to configuration changes without code updates.
+
+## OAuth 2.0's discovery endpoint
+
+Use OAuth 2.0's `oauth-authorization-server` when you're implementing pure OAuth 2.0 (authorization only). Example use cases are
+when you need: - API access tokens - Machine-to-machine authentication - Resource server authorization - Client credentials flow -
+No user identity information needed
```

Comment:
```suggestion
when you need: 
- API access tokens 
- Machine-to-machine authentication 
- Resource server authorization 
- Client credentials flow 
- No user identity information needed
```

The list here is broken - please always check how the page renders either locally before you open the PR or in the preview when it is pushed and in draft stage